src/file.ts

```diff
@@ -1267,12 +1267,10 @@ class File extends ServiceObject<File> {
       ) => {
         if (err) {
           // Get error message from the body.
-          rawResponseStream.pipe(
```

Comment:
In the general sense ... are we making this harder than it needs to be?

```js
const results = [];
rawResponseStream
  .on('data', results.push)
  .on('end', () => {
    err.message = results.concat();
    throughStream.destroy(err);
  });
});
```

Adding @stephenplusplus, the stream whisperer, to chime in :)

Reply:
Can this be a problem with utf-8 when ligatures will be in separate chunk from characters?

Reply:
@TrySound I believe get-stream is doing essentially the example @JustinBeckwith provided, but let me know if I missed it. I think the point is, we could probably duplicate the logic in both of these modules relatively easily and avoid any extra dependencies. In either case, I would still lean towards using get-stream just for simplicity. Especially since it's already a huge win, size-wise, over concat-stream.